src/server/requestHandlers.js

```diff
@@ -168,29 +161,9 @@ fluid.defaults("sjrk.storyTelling.server.saveStoryWithBinariesHandler", {
  * @param {Component} dataSource - an instance of sjrk.storyTelling.server.dataSource.couch.story
  * @param {Boolean} authoringEnabled - a server-level flag to indicate whether authoring is enabled
  */
-sjrk.storyTelling.server.handleSaveStoryWithBinaries = function (request, dataSource, authoringEnabled) {
+sjrk.storyTelling.server.handleSaveStory = function (request, dataSource, authoringEnabled) {
```

Comment:
Rather than having flags at the handler level for `authoringEnabled` we should modify the configuration, probably using context awareness, to not include configuration related to editing, when authoring is disabled.

Reply:
As we discussed, this is out of scope for this Jira. I have filed [SJRK-415](https://issues.fluidproject.org/browse/SJRK-415) to work on this improvement at a later date.